mango-core/src/main/scala/org/bdgenomics/mango/layout/AlignmentRecordLayout.scala

```diff
@@ -167,12 +143,11 @@ object ReadJson {
   /**
    * An implementation of ReadJson which converts AlignmentRecord data to ReadJson
    *
-   * @param recs The list of (Reference, AlignmentRecord) tuples to lay out in json
-   * @param track js track number
+   * @param recs The list of (AlignmentRecord, List[MisMatchJson]) tuples to lay out in json
    * @return List of Read Json objects
    */
-  def apply(recs: List[(ReferenceRegion, AlignmentRecord)], track: Int): List[ReadJson] = {
-    recs.map(rec => new ReadJson(rec._2.getReadName, rec._2.getStart, rec._2.getEnd, rec._2.getReadNegativeStrand, rec._2.getSequence, rec._2.getCigar, rec._2.getMapq, track))
+  def apply(recs: List[(AlignmentRecord, List[MisMatchJson])]): List[ReadJson] = {
```

Comment:
Same as mismatch, let's get rid of the 'json' appending to all of these objects, as they dont contain visual-specific information